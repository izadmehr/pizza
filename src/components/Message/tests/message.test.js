import React from "react";
import { shallow } from "enzyme";

import Message from "../index";

describe("Message Component", () => {
  it("renders with all props", () => {
    const wrapper = shallow(<Message type="success" text="Fake Message" />);
    expect(wrapper).toMatchSnapshot();
  });
});
