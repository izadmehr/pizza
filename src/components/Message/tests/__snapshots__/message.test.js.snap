// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Message Component renders with all props 1`] = `
<styled.div
  type="success"
>
  Fake Message
</styled.div>
`;
